BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -343,6 +384,23 @@ public final class ConvivaAnalytics: NSObject {
         buildDynamicContentMetadata()
     }
 
+    private func buildAdMetadata() -> [String: Any] {
```

Comment:
NIT: Should have some indication in the name that this is for server-side ads.

Reply:
Good point, renamed in 54d8d54